sdk/iot/Azure.Iot.Hub.Service/src/Generated/Models/JobRequestType.cs

```diff
@@ -10,11 +10,7 @@
 
 namespace Azure.Iot.Hub.Service.Models
 {
-    /// <summary>
-    /// Required.
-    /// 
-    /// The type of job to execute.
-    /// </summary>
+    /// <summary> The job type. </summary>
     public readonly partial struct JobRequestType : IEquatable<JobRequestType>
```

Comment:
This looks like a lot of class for something that could almost just be an enum. Has anyone sanity checked this with Pavel or Daniel?

Reply:
But that would mean service change right? They are just making non-breaking changes at the moment.

Reply:
No, it wouldn't be a service change. I'm just talking about how the end user specifies this value. We'd need a string representation of that enum value, which I think there is a json attribute to use (at least there was in Newtonsoft).

Reply:
Follow up with Pavel on this